docs/getting-started/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Getting Started
+description: This section provides a conceptual overview of Octopus Deploy, and links to documentation that guides you through your own self-hosted or cloud-hosted Octopus server.
+position: 1
+---
+
+Welcome!
+
+This section provides an overview of Octopus Deploy concepts and links to the relevant documentation, which explore the concepts further and guides you through implementing them with your own [self-hosted](#self-hosted-octopus) or [cloud-hosted](#octopus-cloud) Octopus server.
+
+
+## How Octopus Works
+
+Octopus Deploy allows users to consistently and reliably deploy by creating a Release and deploying it following the steps defined in a Projects Deployment Process.
```

Comment:
I'm not sure about `reliably deploy by creating`. While it might feel like it's stating the obvious, sometimes I think it makes sense to do so, not all of our users have English as a first language, or if they're just in a hurry and don't want to do a careful reading. You could expand the sentence: `reliably deploy their software by creating releases of the software...`, but I think you cover it in the next paragraph, so it might be better to just delete this opening sentence.